### PR TITLE
New features in kubetest2 openshift

### PR DESCRIPTION
**Pull request by chrislovecnm**
- two new flags that control the logic of the deployer
- first flag can turns off the capability of enabling the required
services in the gcloud project.
- second flag causes the down process to not delete the service
account file
- updated docs to provide details about the above flags
- updated go.mod to the latest kubetest2

### CODE REVIEW

**Review by keith-mcclellan — DISMISSED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -10,19 +10,19 @@ require (
...
+	github.com/onsi/gomega v1.10.3
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.6.1
+	github.com/stretchr/testify v1.7.0
 	go.uber.org/zap v1.15.0
-	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
-	gopkg.in/yaml.v2 v2.3.0
+	golang.org/x/net v0.0.0-20201224014010-6772e930b67b
+	gopkg.in/yaml.v2 v2.4.0
```
Comment (chrislovecnm) [inline go.mod:25]: This update in go.md, I think is modifying the formatting in our CRDs.

**Review by keith-mcclellan — APPROVED** [review]
LGTM

**Comment by rail** [comment]
I think we also need to update the rule in https://github.com/cockroachdb/cockroach-operator/blob/c38b9037fc9ea2f5a7d2b73bc5f153a11d916aee/Makefile#L157

**Pull request merged**